bin/list-all

```diff
@@ -2,7 +2,7 @@
 
 RELEASES_URI=https://raw.githubusercontent.com/dotnet/core/master/release-notes/releases-index.json
 FILE="releases.json"
-KEY="release-version"
+KEY="latest-sdk"
```

Comment:
Why not list all available versions instead of only the latest one?

Reply:
Ok, I see. To match each version, the `match_key` function needs to handle only the `sdk.version` keys, and ignore the `runtime.version` keys. That's tricky, we can handle this later. 👌